apps/tng-dictionary/src/components/VocabularyListIndex/VocabularyListIndex.tsx

```diff
@@ -12,130 +12,216 @@ import stringIncludes from '../../utilities/matchers/stringIncludes';
 import MiniLoading from '../MiniLoading/mini-loading';
 import './VocabularyListIndex.module.css';
 
+type SearchContext = 'name' | 'nameEnglish';
+
+type VocabularyListSummary = HasIdAndName;
+
 type ComponentState = {
-  vocabularyLists: HasIdAndName[];
-  searchContext: 'name'
-}
+    vocabularyLists: VocabularyListSummary[];
+    searchContext: SearchContext;
+    searchText: string;
+    selectedLists: VocabularyListSummary[];
+};
+
+const mapSwitchStateToSearchContext = (isChecked: boolean): SearchContext =>
+    isChecked ? 'nameEnglish' : 'name';
+
+const mapSearchContextToSwitchState = (searchContext: SearchContext): boolean =>
+    searchContext === 'nameEnglish';
+
+// TODO These labels need to come from settings \ config
+const searchContextToPlaceholder: Record<SearchContext, string> = {
+    name: 'Search Tŝilhqot’in',
+    nameEnglish: 'Search English',
+};
 
-const determineSelectedVocabularyLists = (vocabularyLists: HasIdAndName[], filters: Record<string, string>) =>
-  // @ts-ignore
-  vocabularyLists.filter(name => doValuesMatchFilters(name, filters, stringIncludes))
+const determineSelectedVocabularyLists = (
+    vocabularyLists: VocabularyListSummary[],
+    filters: Record<string, string>
+) =>
+    // @ts-ignore
+    vocabularyLists.filter((name) => doValuesMatchFilters(name, filters, stringIncludes));
```

Comment:
This is where you can replace `stringIncludes` with a function called `stringIncludesCaseInsensitive` (or else add an extra optional argument to `stringIncludes` 
```
stringIncludes(...oldArgs, {shouldIgnoreCase: true})
```